doc/api/webstreams.md

```diff
@@ -1422,7 +1422,7 @@ changes:
 added: v17.0.0
 -->
 
-* `format` {string} One of either `'deflate'` or `'gzip'`.
+* `format` {string} One of `'deflate'`, `'deflate-raw'`, or `'gzip'`.
```

Comment:
Can you add added portion to line 1413?

Reply:
Did you mean `changes` section in YAML which starts in line 1421? (sorry - please bear with me) 
Would something like this work:

```yaml
changes:
  - version: v20.0.0
    pr-url: https://github.com/nodejs/node/pull/50097
    description: format now accepts `deflate-raw` value
```

What version number should I use?

Reply:
Yes. Please use REPLACEME as the version number

Reply:
Thanks. Done.